core/app/components/FormBuilder/actions.ts

```diff
@@ -54,13 +54,12 @@ export const saveForm = defineServerAction(async function saveForm(form: FormBui
 			} else if (!element.elementId) {
 				// Newly created elements have no elementId
 				acc.upserts.push(formElementsInitializerSchema.parse({ formId, ...element }));
-			} else if (element.updated || element.order !== index + 1) {
```

Comment:
This was how we reordered elements before on the backend. Now the ranks are updated entirely on the frontend, so we can just remove this.